addons/website/static/src/snippets/s_searchbar/search_bar.js

```diff
@@ -198,16 +201,21 @@ export class SearchBar extends Interaction {
                 this.render();
                 break;
             case "ArrowUp":
+                ev.preventDefault();
+                if (this.menuEl) {
+                    const allResults = [...this.menuEl.querySelectorAll(".o_search_result_item a")];
```

Comment:
```suggestion
                    const resultEls = this.menuEl.querySelectorAll(".o_search_result_item > a");
```
:slightly_frowning_face: Els suffix for var holding element value + querySelectorAll itself should return list + we should target direct anchor child only.